doc/man.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+version=0.1
+date=$(date +"%B %Y")
+
+make_man () {
+    head_name=$(head -n 1 $entry)
+    head_name=${head_name//\# /}
+
+    file=$(tail -n +3 $entry)
+    file=${file//\#\# /\# }
+    file="% $head_name($file_ext) Version 1.0 | OpenVAS User Manual"$'\n'$file
+
+    filename=${filename//.md/.${file_ext}}
+
+    echo "$file" | pandoc --standalone -f markdown -t man -o $man_dir/$filename /dev/stdin
+}
+
+recursive_functions () {
+    for entry in "$search_dir"/*
+    do 
+        # In case of folder iterate through it
+        if [[ -d $entry ]]; then
+            search_dir="$entry"
+            recursive_functions
+        # Else make an entry for the file
+        elif [[ -f $entry ]]; then
+            filename="$(basename -- $entry)"
+            if [ $filename != index.md && $filename ]; then
```

Comment:
I got the following error when I run `make nasl-man`
```
Scanning dependencies of target nasl-man
./man.sh: line 29: [: missing `]'
./man.sh: line 29: [: missing `]'
./man.sh: line 29: [: missing `]'
./man.sh: line 29: [: missing `]'
./man.sh: line 29: [: missing `]'
./man.sh: line 29: [: missing `]'
./man.sh: line 29: [: missing `]'
Built target nasl-man
```

Reply:
This is fixed now